src/main/src/main/java/org/geoserver/catalog/LayerInfo.java

```diff
@@ -66,6 +67,20 @@ enum WMSInterpolation {
      */
     Set<StyleInfo> getStyles();
 
+    /**
+     * The non default styles available for the layer.
+     *
+     * @uml.property name="styles"
+     * @uml.associationEnd multiplicity="(0 -1)" container="java.util.Set"
+     *     inverse="resourceInfo:org.geoserver.catalog.StyleInfo"
+     */
+    default Set<StyleInfo> styles() {
```

Comment:
I do not think we need a UML annotation here - this is a query method that does not store any information on its own. 

You may also find that the QA check requires a return javadoc in order to pass.

Reply:
Updated Javadoc with return param